### ISSUE TITLE
Updating search indexes exception

### ISSUE THREAD

**Issue by rauwebieten** [barrelstrength/craft-sprout-forms#425]
### Description (screenshots, logs, ...)

It seems the queued jobs for reports are triggering an exception.
Error from php:   
```
session_set_cookie_params(): Cannot change session cookie parameters when headers already sent
```

Job data: 
```
{
    "elementType": "barrelstrength\\sproutbasereports\\elements\\Report",
    "elementId": "190514",
    "siteId": "1",
    "fieldHandles": [],
    "description": null
}
```

### Steps to reproduce

1. Create a report
2. Updating search indexes fails.

Looks like this in the admin: (queue manager)
https://pasteboard.co/IVfItS0.png

### Additional info

- Craft version: 3.4.5
- PHP version: 7.3.11
- Plugins & versions: "barrelstrength/sprout-forms": "3.8.2"

**Comment by BenParizek** [comment]
@rauwebieten I can recreate this. Working on a fix.

**Comment by BenParizek** [comment]
@rauwebieten We just released Sprout Forms v3.8.3 which should solve this issue.

**Comment by rauwebieten** [comment]
thank you!
